app/presenters/content_item/updatable.rb

```diff
@@ -14,7 +14,7 @@ def history
     end
 
     def first_public_at
-      content_item["details"]["first_public_at"]
```

Comment:
I think for safety on this you might have to use `content_item["details"]["first_public_at"] || content["first_published_at"]`

Reply:
Updated